package.json

```diff
@@ -188,6 +188,14 @@
           "configuration.entries.item.put"
         ],
         "visible": true
+      },
+      {
+        "permissionName": "ui-developer.settings.okapiTimers",
+        "displayName": "Settings (developer): Can view Okapi timers",
+        "subPermissions": [
+          "okapi.proxy.tenants.timers.list"
+        ],
+        "visible": true
```

Comment:
If you wish to only view timers for current tenant you don't need this permission at all, since Okapi has

    "permissionsRequiredTenant" : [ ]

Same goes for viewing details for a particular timer.

For changes to timers, some permissions is required.

Reply:
The intent, for now, is read-only access, so I will remove the excess permission. Thanks for mentioning this! Lack of visibility into the timers came up during yesterday's release retro, and it seemed like an easy problem to resolve. 

I suspect folks will want a change-timers GUI eventually. We'll add permissions then.